lib/room.cpp

```diff
@@ -2065,7 +2065,7 @@ QJsonObject Room::Private::toJson() const
         for (const auto *m : membersMap)
             appendStateEvent(stateEvents, QStringLiteral("m.room.member"),
                 { { QStringLiteral("membership"), QStringLiteral("join") }
-                , { QStringLiteral("displayname"), m->rawName(q) }
+                , { QStringLiteral("displayName"), m->rawName(q) }
```

Comment:
Nah, that will break things. The loading code will still expect `displayname` because The Spec uses `displayname`.